test/test_python.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+from rpmlint.checks.PythonCheck import PythonCheck
+from rpmlint.filter import Filter
+
+from Testing import CONFIG, get_tested_package
+
+
+@pytest.fixture(scope='function', autouse=True)
+def pythoncheck():
+    CONFIG.info = True
+    output = Filter(CONFIG)
+    test = PythonCheck(CONFIG, output)
+    return output, test
+
+
+@pytest.mark.parametrize('package', ['binary/pythoncheck-python-doc-in-package'])
+def test_python_doc_in_package(tmpdir, package, pythoncheck):
+    output, test = pythoncheck
+    test.check(get_tested_package(package, tmpdir))
+    out = output.print_results(output.results)
+    assert 'W: python-doc-in-package /usr/lib/python2.7/site-packages/python-mypackage/doc' in out
+    assert 'W: python-doc-in-package /usr/lib/python2.7/site-packages/python-mypackage/docs' in out
+    assert 'W: python-doc-in-package /usr/lib/python3.8/site-packages/python-mypackage/doc' in out
+    assert 'W: python-doc-in-package /usr/lib/python3.8/site-packages/python-mypackage/docs' in out
+
+
+@pytest.mark.parametrize('package', ['binary/pythoncheck-python-tests-in-package'])
+def test_python_tests_in_package(tmpdir, package, pythoncheck):
+    output, test = pythoncheck
+    test.check(get_tested_package(package, tmpdir))
+    out = output.print_results(output.results)
+    assert 'W: python-tests-in-package /usr/lib/python2.7/site-packages/python-mypackage/test' in out
+    assert 'W: python-tests-in-package /usr/lib/python2.7/site-packages/python-mypackage/tests' in out
+    assert 'W: python-tests-in-package /usr/lib/python3.8/site-packages/python-mypackage/test' in out
+    assert 'W: python-tests-in-package /usr/lib/python3.8/site-packages/python-mypackage/tests' in out
```

Comment:
Why is this a problem?

Reply:
At least in my day-to-day packaging practice I've always followed a package-tests-in-a-dedicated-subpackage convention. Just like @scarabeusiv pointed out for documentation, it is a waste of space and should rate a warning since tests are usually not necessary for a package's day-to-day operation. Also, here's a quick informal survey for `test` modules on my machine:

```
% ls -l /usr/lib/python3.8/site-packages | wc -l
461
% find /usr/lib/python3.8/site-packages -type d -name test       
/usr/lib/python3.8/site-packages/unittest2/test
/usr/lib/python3.8/site-packages/terml/test
/usr/lib/python3.8/site-packages/mypy/test
/usr/lib/python3.8/site-packages/pypeg2/test
/usr/lib/python3.8/site-packages/pyftpdlib/test
/usr/lib/python3.8/site-packages/ometa/test
/usr/lib/python3.8/site-packages/cheroot/test
/usr/lib/python3.8/site-packages/hyperlink/test
/usr/lib/python3.8/site-packages/mypyc/test
/usr/lib/python3.8/site-packages/ndg/httpsclient/test
/usr/lib/python3.8/site-packages/cherrypy/test
```

I had a quick look at all of these and they are all tests. Now there may be a package out there which does have a `test` module that is not a test, but vitally important in day-to-day operation. But the data I am looking at says such a package is an exception and not the rule. Hence I retained this warning, albeit with the added `-test` check on the package name you suggested below.

Reply:
With the package name test (and suggestion) this makes sense.